locationTrivia-ObjectsTests/FISLocationSpec.m

```diff
@@ -16,99 +10,119 @@
         });
 
-        it(@"Should return YES for a valid location", ^{
-            FISLocation *loc = [[FISLocation alloc] init];
-            loc.name = @"Empire State Building";
-            loc.latitude = @40.7484;
-            loc.longitude = @-73.9857;
-            expect([loc verifyLocation]).to.beTruthy();
+        it(@"should return NO for a latitude greater than 90.0", ^{
+            empireState.latitude = 90.01;
+            expect([empireState validateLocationData]).to.beFalsy();
         });
-
-        it(@"Should return NO for invalid latitude", ^{
-            FISLocation *loc = [[FISLocation alloc] init];
-            loc.name = @"Empire State Building";
-            loc.latitude = @140.7484;
-            loc.longitude = @-73.9857;
-            expect([loc verifyLocation]).to.beFalsy();
+        
+        it(@"should return NO for a latitude less than -90.0", ^{
+            empireState.latitude = -90.01;
+            expect([empireState validateLocationData]).to.beFalsy();
         });
 
-        it(@"Should return NO for invalid longitude", ^{
-            FISLocation *loc = [[FISLocation alloc] init];
-            loc.name = @"Empire State Building";
-            loc.latitude = @40.7484;
-            loc.longitude = @-1173.9857;
-            expect([loc verifyLocation]).to.beFalsy();
+        it(@"should return NO for a longitude greater than 180.0", ^{
+            empireState.longitude = 180.01;
+            expect([empireState validateLocationData]).to.beFalsy();
+        });
+        
+        it(@"should return NO for a longitude less than -180.0", ^{
+            empireState.longitude = -180.01;
+            expect([empireState validateLocationData]).to.beFalsy();
+        });
+        
+        it(@"should return NO for a longitude equal to -180.0", ^{
+            empireState.longitude = -180.0;
+            expect([empireState validateLocationData]).to.beFalsy();
         });
 
-        it(@"Should return NO for invalid name", ^{
-            FISLocation *loc = [[FISLocation alloc] init];
-            loc.name = @"";
-            loc.latitude = @40.7484;
-            loc.longitude = @-73.9857;
-            expect([loc verifyLocation]).to.beFalsy();
+        it(@"should return NO for a name containing an empty string", ^{
+            empireState.name = @"";
+            expect([empireState validateLocationData]).to.beFalsy();
+        });
+        
+        it(@"should return NO for a name set to 'nil'", ^{
+            empireState.name = nil;
+            expect([empireState validateLocationData]).to.beFalsy();
         });
     });
 
-    describe(@"Shorten Location Name", ^{
-        it(@"Should have a method called shortenLocationNameToLength:", ^{
-            expect(location).to.respondTo(@selector(shortenedNameToLength:));
+    describe(@"stringByTruncatingNameToLength:", ^{
+        it(@"should return a 6-letter substring of the location's name when length equals 6", ^{
```

Comment:
Worth adding a check for truncating to a length > the length of the string? I think `-substringToIndex:` crashes in that case.